### PR TITLE
[CES-769] sending gcodes to ultimoco

### PR DESCRIPTION
**Pull request by Jipolie01**
Loading new DMA driver

### CODE REVIEW

**Comment by tjclement** [comment]
Nice work :)

File: configs/sx8m_config [hunk 1]
```diff
@@ -1808,7 +1808,7 @@ CONFIG_TOUCHSCREEN_PROPERTIES=y
 # CONFIG_TOUCHSCREEN_INEXIO is not set
 # CONFIG_TOUCHSCREEN_MK712 is not set
 # CONFIG_TOUCHSCREEN_PENMOUNT is not set
-CONFIG_TOUCHSCREEN_EDT_FT5X06=y
+CONFIG_TOUCHSCREEN_EDT_FT5X06=m
```
[the remark this answers is not in the document]
Comment (Jipolie01) [reply configs/sx8m_config:1811]: Raymond made most of the these changes, so I'm not quite sure what it related to. It could be that these were changes still on the CES-719 branch when he branched off

File: configs/sx8m_config [hunk 2]
```diff
@@ -76,9 +76,7 @@ CONFIG_GENERIC_IRQ_CHIP=y
 CONFIG_IRQ_DOMAIN=y
 CONFIG_IRQ_DOMAIN_HIERARCHY=y
 CONFIG_GENERIC_MSI_IRQ=y
-CONFIG_GENERIC_MSI_IRQ_DOMAIN=y
-CONFIG_HANDLE_DOMAIN_IRQ=y
-# CONFIG_IRQ_DOMAIN_DEBUG is not set
+CONFIG_GENERIC_proptietary_DEBUG is not set
```
Comment (tjclement) [inline configs/sx8m_config:79]: Were these changes intentional in the last commit? I don't know what these options do

**Review by tjclement — APPROVED** [review]
